Build failing due to modbus integration tests

Since several days, the nightly builds and some PR builds are failing due to an error in modbus integration tests.

```
[ERROR] Error   : Exit code remote process 7: /home/jenkins/jenkins-agent1/tools/hudson.model.JDK/OpenJDK_11/bin/java -cp /home/jenkins/jenkins-agent1/workspace/openHAB-Addons/itests/org.openhab.binding.modbus.tests/target/test/tmp/testing/itest/cnf/cache/6.1.0/bnd-cache/biz.aQute.launcher/biz.aQute.launcher.pre.jar -Dlauncher.properties=/home/jenkins/jenkins-agent1/workspace/openHAB-Addons/itests/org.openhab.binding.modbus.tests/target/test/tmp/testing/itest/generated/launch5109827634164306634.properties -Dorg.osgi.service.http.port=45605 --add-opens=java.base/java.io=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.security=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.base/java.time=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.desktop/java.awt.font=ALL-UNNAMED --add-opens=java.naming/javax.naming.spi=ALL-UNNAMED --add-opens=java.rmi/sun.rmi.transport.tcp=ALL-UNNAMED -ea -Dlauncher.runpath=/home/jenkins/jenkins-agent1/workspace/openHAB-Addons/itests/org.openhab.binding.modbus.tests/target/test/tmp/testing/itest/cnf/cache/6.1.0/org.openhab.core.bom.runtime-index/org.eclipse.osgi-3.16.300.v20210525-1715.jar,/home/jenkins/jenkins-agent1/workspace/openHAB-Addons/itests/org.openhab.binding.modbus.tests/target/test/tmp/testing/itest/cnf/cache/6.1.0/bnd-cache/biz.aQute.launcher/biz.aQute.launcher-6.1.0.jar aQute.launcher.pre.EmbeddedLauncher
...
[INFO] openHAB Add-ons :: Integration Tests :: mielecloud Binding Tests SUCCESS [01:23 min]
[INFO] openHAB Add-ons :: Integration Tests :: Modbus Binding Tests FAILURE [ 27.547 s]
[INFO] openHAB Add-ons :: Integration Tests :: Nest Binding Tests SKIPPED
[INFO] openHAB Add-ons :: Integration Tests :: NTP Binding Tests SKIPPED
[INFO] openHAB Add-ons :: Integration Tests :: Systeminfo Binding Tests SKIPPED
[INFO] openHAB Add-ons :: Integration Tests :: TRÅDFRI Binding SKIPPED
[INFO] openHAB Add-ons :: Integration Tests :: Wemo Binding Tests SKIPPED
[INFO] openHAB Add-ons :: Integration Tests :: MapDB Persistence Tests SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  57:52 min (Wall Clock)
[INFO] Finished at: 2022-02-13T04:24:13Z
[INFO] ------------------------------------------------------------------------

[INFO] Static code analysis summary report is available in:
[INFO] file:/home/jenkins/jenkins-agent1/workspace/openHAB-Addons/target/summary_report.html
[ERROR] Failed to execute goal biz.aQute.bnd:bnd-testing-maven-plugin:6.1.0:testing (default) on project org.openhab.binding.modbus.tests: 1 errors found -> [Help 1]
```

--- a/itests/org.openhab.binding.modbus.tests/src/main/java/org/openhab/binding/modbus/tests/ModbusDataHandlerTest.java
+++ b/itests/org.openhab.binding.modbus.tests/src/main/java/org/openhab/binding/modbus/tests/ModbusDataHandlerTest.java
@@ -1066,8 +1066,8 @@ public class ModbusDataHandlerTest extends AbstractModbusOSGiTest {
         dataConfig.put("readStart", "0");
         dataConfig.put("readValueType", "foobar");
         testInitGeneric(ModbusReadFunctionCode.READ_MULTIPLE_REGISTERS, dataConfig, status -> {
-            assertThat(status.getStatus(), is(equalTo(ThingStatus.OFFLINE)));
-            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.CONFIGURATION_ERROR)));
+            assertThat(status.getStatus(), is(equalTo(ThingStatus.UNINITIALIZED)));
+            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.HANDLER_CONFIGURATION_PENDING)));
         });
     }
 
@@ -1111,8 +1111,8 @@ public class ModbusDataHandlerTest extends AbstractModbusOSGiTest {
         dataConfig.put("writeValueType", "int8");
         dataConfig.put("writeType", "holding");
         testInitGeneric(null, dataConfig, status -> {
-            assertEquals(ThingStatus.OFFLINE, status.getStatus(), status.getDescription());
-            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.CONFIGURATION_ERROR)));
+            assertThat(status.getStatus(), is(equalTo(ThingStatus.UNINITIALIZED)));
+            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.HANDLER_CONFIGURATION_PENDING)));
         });
     }
 
@@ -1147,7 +1147,8 @@ public class ModbusDataHandlerTest extends AbstractModbusOSGiTest {
         dataConfig.put("writeType", "holding");
         // OFFLINE since sub-register writes are not supported for other than bit
         testInitGeneric(ModbusReadFunctionCode.READ_MULTIPLE_REGISTERS, dataConfig, status -> {
-            assertEquals(status.getStatus(), ThingStatus.OFFLINE, status.getDescription());
+            assertThat(status.getStatus(), is(equalTo(ThingStatus.UNINITIALIZED)));
+            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.HANDLER_CONFIGURATION_PENDING)));
         });
     }
 
@@ -1209,8 +1210,8 @@ public class ModbusDataHandlerTest extends AbstractModbusOSGiTest {
         dataConfig.put("writeValueType", "int8");
         dataConfig.put("writeType", "holding");
         testInitGeneric(ModbusReadFunctionCode.READ_MULTIPLE_REGISTERS, dataConfig, status -> {
-            assertEquals(ThingStatus.OFFLINE, status.getStatus(), status.getDescription());
-            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.CONFIGURATION_ERROR)));
+            assertThat(status.getStatus(), is(equalTo(ThingStatus.UNINITIALIZED)));
+            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.HANDLER_CONFIGURATION_PENDING)));
         });
     }
 
@@ -1255,8 +1256,8 @@ public class ModbusDataHandlerTest extends AbstractModbusOSGiTest {
         dataConfig.put("writeType", "coil");
         dataConfig.put("writeValueType", "foobar");
         testInitGeneric(ModbusReadFunctionCode.READ_COILS, dataConfig, status -> {
-            assertThat(status.getStatus(), is(equalTo(ThingStatus.OFFLINE)));
-            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.CONFIGURATION_ERROR)));
+            assertThat(status.getStatus(), is(equalTo(ThingStatus.UNINITIALIZED)));
+            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.HANDLER_CONFIGURATION_PENDING)));
         });
     }
 
@@ -1266,8 +1267,8 @@ public class ModbusDataHandlerTest extends AbstractModbusOSGiTest {
         dataConfig.put("writeStart", "0");
         dataConfig.put("writeType", "foobar");
         testInitGeneric(ModbusReadFunctionCode.READ_COILS, dataConfig, status -> {
-            assertThat(status.getStatus(), is(equalTo(ThingStatus.OFFLINE)));
-            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.CONFIGURATION_ERROR)));
+            assertThat(status.getStatus(), is(equalTo(ThingStatus.UNINITIALIZED)));
+            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.HANDLER_CONFIGURATION_PENDING)));
         });
     }
 
@@ -1332,8 +1333,8 @@ public class ModbusDataHandlerTest extends AbstractModbusOSGiTest {
         dataConfig.put("readType", "holding");
         dataConfig.put("readValueType", "foobar");
         testInitGeneric(ModbusReadFunctionCode.READ_COILS, dataConfig, status -> {
-            assertThat(status.getStatus(), is(equalTo(ThingStatus.OFFLINE)));
-            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.CONFIGURATION_ERROR)));
+            assertThat(status.getStatus(), is(equalTo(ThingStatus.UNINITIALIZED)));
+            assertThat(status.getStatusDetail(), is(equalTo(ThingStatusDetail.HANDLER_CONFIGURATION_PENDING)));
         });
     }
 

